linux_os/guide/system/software/sudo/sudo_require_reauthentication/oval/shared.xml

```diff
@@ -1,25 +1,20 @@
 <def-group>
   <definition class="compliance" id="{{{ rule_id }}}" version="1">
     {{{ oval_metadata("'Ensure sudo timestamp_timeout is appropriate - sudo timestamp_timeout") }}}
-    <criteria comment="The timestamp_timeout should be configured" >
+    <criteria comment="The timestamp_timeout should be configured">
       <criterion comment="check configuration in /etc/sudoers" test_ref="test_sudo_timestamp_timeout" />
     </criteria>
   </definition>
 
+  <!-- Define test for sudo timestamp. -->
   <ind:textfilecontent54_test check="all" check_existence="only_one_exists" comment="check correct configuration in /etc/sudoers" id="test_sudo_timestamp_timeout" version="1">
     <ind:object object_ref="obj_sudo_timestamp_timeout"/>
-    <ind:state state_ref="state_sudo_timestamp_timeout" />
   </ind:textfilecontent54_test>
 
   <ind:textfilecontent54_object id="obj_sudo_timestamp_timeout" version="1">
-    <ind:filepath operation="pattern match">^/etc/sudoers(\.d/.*)?$</ind:filepath>
-    <ind:pattern operation="pattern match">^[\s]*Defaults[\s]+timestamp_timeout[\s]*=[\s]*([-]?[\d]+)$</ind:pattern>
-    <ind:instance datatype="int" operation="greater than or equal">1</ind:instance>
+     <ind:filepath operation="pattern match">^/etc/sudoers(\.d/.*)?$</ind:filepath>  
+    <ind:pattern operation="pattern match">^[\s]*Defaults[\s]*timestamp_timeout[\s]*=\s*[0-9]+$</ind:pattern>
```

Comment:
```suggestion
    <ind:pattern operation="pattern match">^[\s]*Defaults[\s]+timestamp_timeout[\s]*=\s*[0-9]+$</ind:pattern>
```

I think that there should be at least a space between the `Defaults` and `timestamp_timeout`